components/Avatar/avatar.css

```diff
@@ -0,0 +1,60 @@
+.avatars-wrapper {
+	align-items: baseline;
+	align-self: center;
+	background-color: var(--white);
+	display: flex;
+	justify-content: center;
+	justify-self: center;
+	margin: 1rem;
+	padding: 1rem;
+	width: 80%;
+}
+.avatar {
+	height: 5rem;
+	margin: 1rem;
+	object-fit: cover;
+	width: 5rem;
+}
+.avatar-img {
+	height: 100%;
+	width: 100%;
+}
+.avatar, .avatar-img {
+	border-radius: 50%;
+}
+.avatar-xs {
+	height: 3.5rem;
+	width: 3.5rem;
+}
+.avatar-sm {
+	height: 5rem;
+	width: 5rem;
+}
+.avatar-md {
+	height: 6.5rem;
+	width: 6.5rem;
+}
+.avatar-lg {
+	height: 8rem;
+	width: 8rem;
+}
+.avatar-text {
+	align-items: center;
+	background-color: var(--primary-color);
+	color: var(--secondary-color);
+	display: flex;
+	font-weight: 600;
+	justify-content: center;
```

Comment:
try not to repeat your code make a separate flex class and use it whereever you need

Reply:
I will keep that in mind going forward. I am planning to refactor the code with more utility classes after the completion of the project gradually and I will do so at that time. Thanks for the suggestion.